From eric.dumazet@gmail.com  Thu Jun 23 15:40:24 2011
From: Eric Dumazet <eric.dumazet@gmail.com>
Date: Tue, 30 Mar 2010 05:02:15 +0000
Subject: [PATCH] net: remove redundant code
Git-commit: b05b7d9563f11bf3d7b7f3f53cd74cbfab107355
Patch-mainline: v2.6.35-rc1
References: FATE#311448, bnc#697255

eth_type_trans(skb, netdev) does the "skb->dev = netdev;"
initialization, we can remove it from various network drivers.

Signed-off-by: Eric Dumazet <eric.dumazet@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>

Acked-by: Jeff Mahoney <jeffm@suse.com>
---
 drivers/net/benet/be_main.c |    1 -
 1 file changed, 1 deletion(-)
--- a/drivers/net/benet/be_main.c
+++ b/drivers/net/benet/be_main.c
@@ -826,7 +826,6 @@ static void be_rx_compl_process(struct b
 
 	skb->truesize = skb->len + sizeof(struct sk_buff);
 	skb->protocol = eth_type_trans(skb, adapter->netdev);
-	skb->dev = adapter->netdev;
 
 	vlanf = AMAP_GET_BITS(struct amap_eth_rx_compl, vtp, rxcp);
 	vtm = AMAP_GET_BITS(struct amap_eth_rx_compl, vtm, rxcp);
